backbone.js

```diff
@@ -979,14 +979,14 @@
     sort: function(options) {
       if (!this.comparator) throw new Error('Cannot sort a set without a comparator');
       options || (options = {});
+      var comparator = _.isFunction(this.comparator) ? _.bind(this.comparator, this) : this.comparator;
```

Comment:
I'm *ok* with not handling the `context` case for `iteratee` in lodash since no `iteratee` based function in lodash accepts a `context` param. Users are free to customize `iteratee` to accept a `context` param if they want to expand it for their use.